packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -1,2 +1,142 @@
-## The CDK Construct Library for AWS Elastic Container Service (ECS)
-This module is part of the [AWS Cloud Development Kit](https://github.com/awslabs/aws-cdk) project.
+## AWS Elastic Container Service (ECS) Construct Library
+
+This package contains constructs for working with **AWS Elastic Container
+Service** (ECS). The simplest example of using this library looks like this:
+
+```ts
+// Create an ECS cluster (backed by an AutoScaling group)
+const cluster = new ecs.EcsCluster(this, 'Cluster', {
+  vpc,
+  size: 3,
+  instanceType: new InstanceType("t2.xlarge")
```

Comment:
`instanceCount` is more descriptive than size.

I also think this API for just specifying instance type and count is too simplistic, and limits future options. For example, one nice thing we could do as a TODO is allow people to specify multiple instance types, and whether they want to use spot etc. The ECS L2 construct could then abstract away a lot of the complexity of creating this kind of diverse-bidding-strategy setup. 

As above, I'd favour an `instanceConfiguration` type prop that could allow expressing more advanced configurations.